spec/requests/packages_spec.rb

```diff
@@ -257,6 +257,21 @@
     it 'returns a valid visibility reason' do
       expect(json.data.attributes.visibilityData.reason).to eq 'Set by system'
     end
+
+    describe 'with an invalid url missing package id' do
```

Comment:
Really minor comment @elrickvm : can we rephrase this test name to something else -- `with an invalid url missing package id` is a little confusing because we can pass `url` for certain requests in request body -- probably rename it to something like `getting a package with invalid package id`?